packages/core/core/src/requests/AssetGraphRequest.js

```diff
@@ -593,6 +599,12 @@ export class AssetGraphBuilder {
           }
         }
       }
+      // Sort usedSymbolsUp so they are a consistent order across builds.
+      // This ensures a consistent ordering of these symbols when packaging.
+      // See https://github.com/parcel-bundler/parcel/pull/8212
+      for (let dep of changedDeps) {
+        dep.usedSymbolsUp = new Set([...dep.usedSymbolsUp].sort());
+      }
```

Comment:
Does this intentionally happen inside the `propagateSymbolsUp(...)` callback? This would be called for every node.

Reply:
Moved this out of the callback and verified that there were no significant regressions in build times on a large app 🙂